zerver/webhooks/patreon/fixtures/members_create.json

```diff
@@ -0,0 +1,384 @@
+        "default_country_code": null,
+        "discord_id": null,
+        "email": "patreon@zulip.com",
+        "facebook": null,
+        "facebook_id": null,
+        "first_name": "Zulip",
+        "full_name": "Zulip",
+        "gender": 0,
+        "google_id": "109741564096988987897",
+        "has_password": true,
+        "image_url": "https://c10.patreonusercontent.com/3/eyJ3IjoyMDB9/patreon-media/p/user/7498102/94558813b9354e71a0b4273dc640cc4a/2.png?token-time=2145916800&token-hash=ltapYDjt9zAtefjX41OSe_tCBRarAI24QBf2eMHnjs8%3D",
+        "is_deleted": false,
+        "is_email_verified": true,
+        "is_nuked": false,
+        "is_suspended": false,
+        "last_name": "",
+        "patron_currency": "USD",
+        "social_connections": {
+          "deviantart": null,
+          "discord": null,
+          "facebook": null,
+          "google": null,
+          "instagram": null,
+          "reddit": null,
+          "spotify": null,
+          "twitch": null,
+          "twitter": null,
+          "youtube": null
+        },
+        "thumb_url": "https://c10.patreonusercontent.com/3/eyJ3IjoyMDB9/patreon-media/p/user/7498102/94558813b9354e71a0b4273dc640cc4a/2.png?token-time=2145916800&token-hash=ltapYDjt9zAtefjX41OSe_tCBRarAI24QBf2eMHnjs8%3D",
+        "twitch": null,
+        "twitter": "zulip",
+        "url": "https://www.patreon.com/zulip",
+        "vanity": "zulip",
+        "youtube": null
+      },
+      "id": "7498102",
+      "relationships": {
+        "campaign": {
+          "data": {
+            "id": "1194966",
+            "type": "campaign"
+          },
+          "links": {
+            "related": "https://www.patreon.com/api/campaigns/1194966"
+          }
+        }
+      },
+      "type": "user"
+    },
+    {
+      "attributes": {
+        "amount": 0,
+        "amount_cents": 0,
+        "created_at": null,
+        "description": "Everyone",
+        "patron_currency": "USD",
+        "remaining": 0,
+        "requires_shipping": false,
+        "url": null,
+        "user_limit": null
+      },
+      "id": "-1",
+      "type": "reward"
+    },
+    {
+      "attributes": {
+        "amount": 1,
+        "amount_cents": 1,
+        "created_at": null,
+        "description": "Patrons Only",
+        "patron_currency": "USD",
+        "remaining": 0,
+        "requires_shipping": false,
+        "url": null,
+        "user_limit": null
+      },
+      "id": "0",
+      "type": "reward"
+    },
+    {
+      "attributes": {
+        "amount": 500,
+        "amount_cents": 500,
+        "created_at": "2021-04-07T22:29:15.334+00:00",
+        "currency": "USD",
+        "description": "Contribute to Zulip's development and free hosting for open source projects and other worthy organizations!",
+        "discord_role_ids": null,
+        "edited_at": "2021-04-07T22:29:15.334+00:00",
+        "image_url": null,
+        "patron_amount_cents": 500,
+        "patron_count": 3,
```

Comment:
Just to add a further comment to @timabbott's comment about saying how many you currently have. looks like that date is here, and I definitely agree that's a good idea. If anything that's probably the most important piece of data we can provide. Although beware, that this is a separate field for each patreon reward level, so there will have to be sum sort of sum involved to calculate total.